apps/melding-form/src/app/(general)/samenvatting/Summary.tsx

```diff
@@ -47,12 +48,10 @@ export const Summary = ({ attachments, primaryForm, additionalQuestions, locatio
         </Alert>
       )}
 
-      <Heading className="ams-mb-s" level={1}>
-        {t('step.title')}
-      </Heading>
+      <FormHeader title={t('title')} step={t('step')} />
 
-      <Heading level={2} size="level-4" className="ams-mb-s">
-        {t('title')}
+      <Heading level={1} size="level-4" className="ams-mb-s">
+        {t('main-title')}
```

Comment:
On the summary page the terms are the same size as the H1, which results in a very unclear page.